docs/source/data/subscriptions.mdx

```diff
@@ -540,6 +562,20 @@ The following are community-created `PubSub` libraries for popular event-publish
 
 If none of these libraries fits your use case, you can also create your own `PubSubEngine` subclass. If you create a new open-source library, click **Edit on GitHub** to let us know about it!
 
-## The `graphql-ws` transport library
+## Switching from `subscriptions-transport-ws`
+
+>If you use subscriptions with Apollo Client you must ensure **[both your client and server subscription protocols are consistent](https://www.apollographql.com/docs/react/data/subscriptions/#choice-of-subscription-protocol)** for the subscription library you are using.
+
+This article previously demonstrated using the `subscriptions-transport-ws` library to set up subscriptions. However, this library is no longer actively maintained. You can still use it with Apollo Server, but we strongly recommend using [`graphql-ws`](https://www.npmjs.com/package/graphql-ws) instead.
+
+If you intend to switch from `subscriptions-transport-ws` to `graphql-ws` you will need to:
+* [Adjust Apollo Studio Explorer settings](https://www.apollographql.com/docs/studio/explorer/additional-features/#subscription-support)
+* Update to [Apollo Client 3.6 or later](https://www.apollographql.com/docs/react/data/subscriptions/#choice-of-subscription-protocol)
```

Comment:
@rkoron007 It's actually 3.5.10, which is out of today!

(Apollo iOS should be close to done as well and then we can move forward with this PR!)